templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -656,6 +637,10 @@ Conditions:
     !And [Condition: IsVersion5X, !Or [Condition: IsVersionX0, Condition: IsVersionX1, Condition: IsVersionX2, Condition: IsVersionX3, Condition: IsVersionX4, Condition: IsVersionX5, Condition: IsVersionX6]]
   RestoreRDSOrStandby:
     !Or [Condition: RestoreFromRDSSnapshot, Condition: StandbyMode]
+  CreateESBucket:
```

Comment:
Shouldn't we also remove the `CreateBucket` parameter? This gives an impression that there is a mechanism that creates elasticsearch snapshots.